build/post-integration-test-docker.sh

```diff
@@ -35,12 +46,16 @@ else
 fi
 
 echo "Gathering integration test output"
-cp -r ${WORKSPACE}/fhir-server-test/target/surefire-reports/* ${it_results}/fhir-server-test
+if [ -e ${WORKSPACE}/fhir-server-test/target/surefire-reports ]; then
+    cp -r ${WORKSPACE}/fhir-server-test/target/surefire-reports/* ${it_results}/fhir-server-test
+fi
+
+# Source the tenant1 datastore variables; not strictly needed here
+# but avoids "variable is not set" docker compose warnings
+. ${WORKSPACE}/build/common/set_tenant1_datastore_vars.sh
 
 echo "Bringing down the fhir server docker container(s)..."
-cd ${DIR}/docker
-docker-compose down
+cd ${DIR}/${1}
```

Comment:
might be worth capturing `${1}` in a variable and checking its value before using it. Also, I notice our scripts aren't quoting any of the path arguments, so this assumes we won't have any spaces anywhere. That's probably a reasonable restriction though.

Reply:
any specific recommendation for "checking its value before using it"?

we do check that its nonnull/nonempty at the top:
```
if [ -z ${1} ]; then
  echo "This script requires an argument that points to one of the integration folders"
  exit 1
fi
```

maybe we should prevent `..` in there as well?  anything else?

Reply:
I ended up adding this test `[ -n "$(find ${DIR} -maxdepth 1 -name ${1} -type d)" ]` which is possibly over-restrictive but should be a bit safer

Reply:
You shouldn't need to make that a find. I think you can just check it exists and is a directory (but it looks like the find is doing something similar).
````
if [ ! -d "${DIR}" ]; then
   ...
   exit
fi
````

Reply:
the find ensure that its a subdirectory of ${DIR} that we find.
it could have been 
```
if [ ! -d "${DIR}/${1}" ]; then
   ...
   exit
fi
```

but then ${1} could have `..`

like i said, probably overkill, but i think its ok